sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/DebeziumIO.java

```diff
@@ -38,8 +44,47 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import javax.sql.DataSource;
+
 import static org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Preconditions.checkArgument;
 
+/**
+ * {@link PTransform}s for reading from <a
+ * href="https://debezium.io/documentation/reference/connectors/index.html">Debezium</a> connector.
+ * 
+ * <p>DebeziumIO source returns a unbounded collection of {@code T} as a {@code PCollection<T>}. T is the
+ * type returned by the provided {@link SourceRecordMapper}.
+ * 
+ * <p>To configure the DebeziumIO source, you have to provide a {@link ConnectorConfiguration} using<br>
+ * {@link ConnectorConfiguration#create()}.
+ * Optionally, {@link DataSourceConfiguration#withUsername(String)} and {@link
+ * DataSourceConfiguration#withPassword(String)} allows you to define username and password.
+ *
+ * <p>For example:
+ *
+ * <pre>{@code
+ * PipelineOptions options = PipelineOptionsFactory.create();
+ * Pipeline p = Pipeline.create(options);
+ *       p.apply(DebeziumIO.<String>read().
+ *               withConnectorConfiguration(
+ *                       DebeziumIO.ConnectorConfiguration.create()
+ *                               .withUsername("sa")
+ *                               .withPassword("Password!")
+ *                               .withConnectorClass(SqlServerConnector.class)
+ *                               .withHostName("127.0.0.1")
+ *                               .withPort("1433")
+ *                               .withConnectionProperty("database.dbname", "testDB")
+ *                               .withConnectionProperty("database.server.name", "server1")
+ *                               .withConnectionProperty("table.include.list", "dbo.customers")
+ *                               .withConnectionProperty("include.schema.changes", "false"))
+ *               .withFormatFunction(new SourceRecordJson.SourceRecordJsonMapper())
+ *               .withCoder(StringUtf8Coder.of())
+ *       );
+ * }</pre>
+ * 
+ * Make sure Debezium connector is present in the classpath.
+ *
+ */
```

Comment:
Should we stick with this one or should we use [this](https://github.com/wizeline/beam/pull/9) ?

Reply:
I think this one is more useful because include an example.